python/utils.py

```diff
@@ -26,7 +26,8 @@
 
 """
 
-from qgis.PyQt.QtCore import QCoreApplication, QLocale, QThread, qDebug
+from qgis.PyQt.QtCore import QCoreApplication, QLocale, QThread, qDebug, QUrl
```

Comment:
```suggestion
from qgis.PyQt.QtCore import QCoreApplication, QLocale, QThread, qDebug
```

This is no longer used?

Reply:
Yes it is! I had to be too tired to write a clean line yesterday by night so I mixed up the brushes. Fixed. Thanks!

Ref: <https://doc.qt.io/qt-5/qdesktopservices.html#openUrl>